data/json/items/fuel.json

```diff
@@ -147,7 +147,7 @@
     "effects": [ "FLAME", "STREAM", "INCENDIARY", "NEVER_MISFIRES" ]
   },
   {
-    "id": "lamp_oil",
+    "id": "kerosene",
```

Comment:
I'm not sure about `AMMO` type, but changing `id` of most items breaks saves. You might need to apply a migration for `lamp_oil`.

Reply:
Ouh, I see. What do mean exactly by migration? 
I presume that was the reason why it was left as "lamp oil" instead of kerosene to begin with, huh?

Reply:
Migration is used for items which shouldn't appear in the game anymore. If migration is set for some item, it will be automatically converted to another item on game load.
See `json\items\migration.json` for examples.

Reply:
Ah, great, thank you. So just something like this?
```
{
    "type": "MIGRATION",
    "id": "lamp_oil",
    "replace": "kerosene"
 }
```

Reply:
Yep.